### ISSUE TITLE
Button alignment in landing page is inconsistent.

### ISSUE THREAD

**Issue by Mvishal123** [json-schema-org/website#355]
#### Describe the bug
The button text is not aligned properly in the page. Centering them will make them more visually appealing. 

#### How to Reproduce
Landing page: https://json-schema.org/

![before](https://github.com/json-schema-org/website/assets/113995907/82382781-bd8d-48d7-8701-5d780caea112)

https://github.com/json-schema-org/website

pages/index.page.tsx

#### Expected behavior
The button text is not aligned properly in the page. Centering them will make them more visually appealing.

**Comment by benjagm** [comment]
Do you mean vertical alignment?

**Comment by Michael-Obele** [comment]
Hey @benjagm I believe he means the text align and the padding (space) just below the "getting started" and "join slack" are off compared to "search." Although I have put out a request for a different issue, this seems like a simple bug that I can handle really quickly.

**Comment by MonalSD** [comment]
hey @benjagm if this bug isn't assigned yet can i resolve it quickly ..

**Comment by Mvishal123** [comment]
I've already created a pull request. But the commit is linked with another PR. I also tried revert but I'm unable to remove the commit from another PR. Can anyone guide me through this.

Thanks

**Comment by DarhkVoyd** [comment]
Hello everyone, 

This issue is currently in `Status: Triage`. 
Kindly wait for a maintainer to mark issues to `Status: Available` before creating PRs. Please refer to this [guide](https://github.com/json-schema-org/.github/blob/a88310a03dbb5b21ea5def19613aaf2226572680/TRIAGE.md#L4).

> You are not competing ... you are collaborating.

**Comment by Michael-Obele** [comment]
> Hello everyone,
> 
> This issue is currently in `Status: Triage`. Kindly wait for a maintainer to mark issues to `Status: Available` before creating PRs. Please refer to this [guide](https://github.com/json-schema-org/.github/blob/a88310a03dbb5b21ea5def19613aaf2226572680/TRIAGE.md#L4).
> 
> > You are not competing ... you are collaborating.

I completely agree, and I apologize for asking to handle it while it was still in triage. I now understand the process, and I apologize for the way I phrased my suggestion, which may have come across as dismissive of the triage process.

**Comment by benjagm** [comment]
we have this issue open for the buttons #345 in progress by @DhairyaMajmudar . Maybe he can incorporate this fix at the same time. Can you please collaborate?

Let's make sure left, right, top and bottom paddings/margins are the same for the 3 buttons. The search button is the only one with content text + image together so special consideration will be required here.

**Comment by DhairyaMajmudar** [comment]
Sure I would be happy to incorporate these changes in my existing PR.
